test-app/tests/core/rendering-test.gts

```diff
@@ -9,29 +9,29 @@ import { cell, resource, resourceFactory } from 'ember-resources';
 module('Core | Resource | rendering', function (hooks) {
   setupRenderingTest(hooks);
 
-  module('cleanup with wrapping factory/blueprint', function() {
+  module('cleanup with wrapping factory/blueprint', function () {
     test('a generated interval can be cleared', async function (assert) {
       const id = cell(0);
       const condition = cell(true);
 
-      const poll = resourceFactory((id: number) => {
```

Comment:
resourceFactory isn't needed here, and is kinda awkward to have inline like this (moreso that the call below, imo)